docs/src/main/paradox/routing-dsl/source-streaming-support.md

```diff
@@ -137,12 +134,20 @@ In this example, we want to consume this data in a streaming fashion from the re
 back pressure to the underlying TCP connection should the server be unable to cope with the rate of incoming data. Back pressure
 is automatically applied thanks to @extref[Akka Streams](akka-docs:scala/stream/index.html).
 
+Scala
```

Comment:
Shouldn't we add here the snippet from `measurement-model`? Now that is removed from the top section is never showed to the reader